app/validators/accredited_provider_id_format_validator.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AccreditedProviderIdFormatValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
```

Comment:
I think we already had some validation logic for this? @defong can you confirm

Reply:
We have a few format specs, maybe you are thinking of `spec/validators/ukprn_format_validator_spec.rb` ?

Reply:
Oh I just done an inline validation

https://github.com/DFE-Digital/publish-teacher-training/blob/b331261175e0b5a2a1ee9ca857a442fd8094161a/app/forms/support/provider_form.rb#L80-L95

Reply:
I think we have to allow blank on update.

Reply:
I like that regex!